lib/Settings/Section.php

```diff
@@ -0,0 +1,60 @@
+<?php
```

Comment:
There is a mix of tabs and spaces used for indentation in this file and most of the tabs have a length of 8, could you get that unified to be all tabs of length 4.